airbyte-integrations/connectors/source-stripe/source_stripe/streams.py

```diff
@@ -55,9 +55,15 @@ def __init__(self, cursor_field: str, legacy_cursor_field: Optional[str]):
     def extract_records(self, response: requests.Response) -> Iterable[MutableMapping]:
         records = super().extract_records(response)
         for record in records:
-            if self.legacy_cursor_field and self.cursor_field not in record:
-                record[self.cursor_field] = record[self.legacy_cursor_field]
-            yield record
+            if self.cursor_field in record:
+                yield record
+                continue  # Skip the rest of the loop iteration
```

Comment:
nit: Would it be more readable to use the `else` statement instead of having to add a comment?

Reply:
matter of taste :) I prefer early exit/continue over else statements

Reply:
but if you insist, I can change it of course

Reply:
There are a couple of resources regarding cognitive complexity recommending to avoid "jumps to" labels. One example is [Sonar](https://www.sonarsource.com/docs/CognitiveComplexity.pdf?_gl=1*rr0zbd*_gcl_au*NDY5NzI2ODIxLjE2OTU0MDUxMDQ.) I personally agree with these but I'll not enforce it if the maintainers of the code prefer otherwise

Reply:
Thank you! I will leave it here as is, but take into account in the future updates